cinch/roles/jenkins_master/defaults/main.yml

```diff
@@ -21,6 +21,8 @@ https_enabled: false
 # to construct the name of the RPM below as well as update center URLS, so this
 # must include the complete "x.y.z" version, e.g. 1.651.3.
 jenkins_version: "1.651.3"
+# Remove jenkins package from the upgrade_blacklist variable to allow upgrades
```

Comment:
We might want to say "Temporarily ignore presence of jenkins in the upgrade_blacklist..." or such, in order to avoid confusion that this might edit the local Ansible variables on-disk or pull some other wonkiness.